class.jetpack-gutenberg.php

```diff
@@ -556,4 +556,32 @@ public static function enqueue_block_editor_assets() {
 
 		wp_enqueue_style( 'jetpack-blocks-editor', $editor_style, array(), $version );
 	}
+
+	/**
+	 * Some blocks do not depend on a specific module,
+	 * and can consequently be loaded outside of the usual modules.
+	 * We will look for such modules in the extensions/ directory.
+	 *
+	 * @since 7.1.0
+	 */
+	public static function load_independent_blocks() {
+		// Bail early if Blocks should not be loaded on the site.
+		if (
+			! self::should_load()
+			|| ! self::is_gutenberg_available()
+		) {
+			return;
+		}
+
+		/**
+		 * Look for files that match our list of available Jetpack Gutenberg extensions (blocks and plugins).
+		 * If available, load them.
+		 */
+		foreach ( self::$extensions as $extension ) {
+			$extension_file_glob = glob( JETPACK__PLUGIN_DIR . 'extensions/*/' . $extension . '/' . $extension . '.php' );
+			if ( ! empty( $extension_file_glob ) ) {
+				include_once $extension_file_glob[0];
+			}
+		}
```

Comment:
There's something that feels odd when functions return something on one branch and nothing on another. This could written as:
```php
public static function load_independent_blocks() {
    if ( self::should_load && self::is_gutenberg_available ) ) {
        foreach ( self::$extensions as $extension ) {
            ...
        }
    }
```

Reply:
You're right, it does look better that way. Updated in 1f19d8b2d33c4a136b6916e8d390ad2d2c07b2fd